lib/tessel/deployment/rust.js

```diff
@@ -1,10 +1,17 @@
 // System Objects
 var path = require('path');
+var http = require('http');
+var path = require('path');
 
 // Third Party Dependencies
 var fs = require('fs-extra');
 var tags = require('common-tags');
 var toml = require('toml');
+require('toml-require').install();
```

Comment:
Where is this used and why? Presently, the module `toml` is handling the parsing of toml files: https://github.com/tessel/t2-cli/blob/a7c32f6eeaf74c6f616b84c4db005e93b4c3cd13/lib/tessel/deployment/rust.js#L30-L36